docs/production/install.md

```diff
@@ -1,31 +1,42 @@
-[the installation requirements](requirements.md). Alternatively,
-you can use a preconfigured
-[DigitalOcean droplet](https://marketplace.digitalocean.com/apps/zulip?refcode=3ee45da8ee26), or
-Zulip's
-[experimental Docker image](deployment.md#zulip-in-docker).
+- Follow these instructions to **install a self-hosted Zulip server on a system
+  of your choice**.
+- Use a preconfigured
+  [DigitalOcean droplet](https://marketplace.digitalocean.com/apps/zulip?refcode=3ee45da8ee26)
+- Use Zulip's [experimental Docker image](deployment.md#zulip-in-docker).
+- Use [Zulip Cloud](https://zulip.com/plans/) hosting. Read our [guide to choosing between Zulip Cloud and
+  self-hosting](https://zulip.com/help/getting-your-organization-started-with-zulip#choosing-between-zulip-cloud-and-self-hosting).
 
-### Should I follow this installation guide?
+To **import data** from [Slack][slack-import], [Mattermost][mattermost-import], [Rocket.Chat][rocketchat-import], [Gitter][gitter-import], [Zulip Cloud][zulip-cloud-import], or [another Zulip
+server][zulip-server-import], follow the linked instructions.
 
-- If you would like to try out Zulip, you can start by [checking it out in the
-  Zulip development community](https://zulip.com/try-zulip/), or [create a test
-  Zulip Cloud organization](https://zulip.com/new/).
+You can **try out Zulip** before setting up your own server by [checking
+it out](https://zulip.com/try-zulip/) in the Zulip development community, or
+[creating a free test organization](https://zulip.com/new/) on Zulip Cloud.
 
-- If you are deciding between self-hosting Zulip and signing up for [Zulip
-  Cloud](https://zulip.com/plans/), our [self-hosting
-  overview](https://zulip.com/self-hosting/) and [guide to choosing between
-  Zulip Cloud and
-  self-hosting](https://zulip.com/help/getting-your-organization-started-with-zulip#choosing-between-zulip-cloud-and-self-hosting)
-  are great places to start.
+:::{note}
+These instructions are for self-hosting Zulip. To
+[contribute](../contributing/contributing.md) to the project, set up the
+[development environment](../development/overview.md).
+:::
 
-- If you're developing for Zulip, you should follow the instructions
-  to install Zulip's [development environment](../development/overview.md).
+## Installation process overview
 
-If you'd like to install a self-hosted Zulip server, this guide is for you!
+0. [Configure your server](#step-0-configure-your-server)
+1. [Download the latest release](#step-1-download-the-latest-release)
+1. [Install Zulip](#step-2-install-zulip)
+1. [Create a Zulip organization, and log in](#step-3-create-a-zulip-organization-and-log-in)
+
+That's it! Once installation is complete, you can
+[configure](settings.md) Zulip to suit your needs.
+
+## Step 0: Configure your server
+
+Configure an Ubuntu or Debian system that satisfies [the installation
+requirements](requirements.md) for your expected usage level.
```

Comment:
Maybe something more like "Provision and log in to a fresh Ubuntu or Debian system in your preferred hosting environment that satisfies the installation requirements for your expected usage level.". Key points are:
* Hint that it should be a fresh server, not one running other stuff. We might even want to add a reminder that we want an empty system (of applications) but it's fine to have your standard monitoring/etc. software on the system.
* Spell out that you should ssh into it.